packages/document/module-doc/docs/en/guide/faq/build.mdx

```diff
@@ -212,7 +212,7 @@ For source code files, it is recommended to fix the type problem, which can bett
 
 1. enable [skipLibCheck](https://www.typescriptlang.org/tsconfig#skipLibCheck) to skip the d.ts check of the thrid-party package.
 2. If the package exports ts files directly, skipLibCheck will not work because it can only skip the d.ts check,
-so you can only turn off [dts.abortOnError](/api/config/build-config.html#dtsabortonerror) to ignore these errors.
+   so you can only turn off [dts.abortOnError](/api/config/build-config.html#dtsabortonerror) to ignore these errors.
```

Comment:
should we add an empty line before this section

Reply:
why

Reply:
Or maybe should be in the same line?
![image](https://github.com/web-infra-dev/modern.js/assets/7237365/5fc333c2-8b8f-4eb2-acde-8f0c5b8243e2)

Reply:
yes,they are in the same line before.